imposter.go

```diff
@@ -101,6 +125,33 @@ func (r TCPRequest) toDTO() tcpRequestDTO {
 	return dto
 }
 
+// fromMapValues parses a mountebank JSON representation of an HTTP header or query
+// into its respective stdlib map representation.
+func fromMapValues(q map[string]interface{}) (map[string][]string, error) {
+	out := make(map[string][]string, len(q))
+
+	for k, v := range q {
+		switch typ := v.(type) {
+		case string:
+			out[k] = []string{typ}
+		case []interface{}:
+			ss := make([]string, len(typ))
+			for i, elem := range typ {
+				s, ok := elem.(string)
+				if !ok {
+					return nil, errors.New("invalid query key array subtype")
+				}
+				ss[i] = s
+			}
+			out[k] = ss
+		default:
+			return nil, errors.New("invalid query key type")
```

Comment:
Would be helpful to put `typ` in the error message

Reply:
Fixed the typo - https://github.com/senseyeio/mbgo/pull/11/commits/ab886ca44857f53c76e921341a9dc080064a7fb3.